src/features/_REFACTOR.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
Toto by som asi necommitoval

Reply:
jo, sorry, to boli moje poznamky, zmazem